release_notes_generator/model/record.py

```
@@ -21,7 +21,7 @@
 import logging
 import re
 
-from typing import Optional, AnyStr, Any
+from typing import Optional, AnyStr, Any, Dict, List
```

Ruff (0.11.9)

`typing.AnyStr` imported but unused

Remove unused import: `typing.AnyStr`

(F401)